### DISCUSSION TITLE
Requesting join team

### DISCUSSION THREAD

**Discussion by IdkAnythin07** [ROM-builders/temporary#19357] [General]
Github: https://github.com/IdkAnythin07
DT: https://github.com/IdkAnythin07/android_device_realme_RMX1941
Telegram: https://t.me/IdkAnythin07

PR:
#18932 

Successful Builds:
Cirrus: https://cirrus-ci.com/github/ROM-builders/temporary/pull%
DT: https://github.com/IdkAnythin07/android_device_realme_RMX1941/tree/bliss
Local Manifest: https://github.com/idkAnythin07/local_manifest/tree/bliss
PR: https://github.com/IdkAnythin07/temporary/tree/bliss
I also saw #learn and I know git-cli, cherry pick, commit

**Comment by IdkAnythin07** [comment]
@Apon77 sir?

**Comment by IdkAnythin07** [comment]
@Apon77 sir?

**Comment by Apon77** [comment]
Please follow #learn instruction from telegram group
